src/runtime/rpc/network.cpp

```diff
@@ -442,6 +457,18 @@ void rpc_session::start_negotiation()
         }
 
         auth_negotiation();
+    } else {
+        // set negotiation success if auth is disabled
+        complete_negotiation(true);
```

Comment:
For boolean parameter, better to add some comments on it, alternatively, use enum instead of bool, https://abseil.io/tips/94

Reply:
I have add some comments in front of the interface `complete_negotiation` in network.h